components/enrichers/reachability/task.yaml

```diff
@@ -18,15 +18,19 @@ spec:
       description: The workspace containing the source-code to scan.
   steps:
   - name: run-cdxgen
-    image: ghcr.io/cyclonedx/cdxgen:v10.9.5
+    image: ghcr.io/cyclonedx/cdxgen:v11.1.0
     command:
       - node
+    env:
+      - name: NPM_INSTALL_ARGS
+        value: "--force --ignore-scripts --package-lock --legacy-peer-deps"
     args:
       - /opt/cdxgen/bin/cdxgen.js
       - -t
       - $(params.enricher-reachability-programming-language)
       - --deep
       - -r
+      - --feature-flags safe-pip-install
```

Comment:
I guess we need this only if python is the supplied lang?

Reply:
surprisingly it won't fail even if python isn't the supplied lnag which saves me an extra if statement